### ISSUE TITLE
Type error fixes cause errors in next.js

### ISSUE THREAD

**Issue by jaywcjlove** [timlrx/rehype-prism-plus#65]
@timlrx Type error fixes cause errors in next.js

- https://github.com/uiwjs/react-md-editor/issues/561

_Originally posted by @jaywcjlove in https://github.com/timlrx/rehype-prism-plus/issues/64#issuecomment-1667103879_

https://github.com/timlrx/rehype-prism-plus/blob/d8582ab809420b463f57782e7f67dadae488207b/package.json#L13-L29

The reason for the error is: `import`+`require` must be used in combination in the exports field in next.js.

**Comment by jaywcjlove** [comment]
```tree
.
├── all.d.ts
├── all.es.js
├── all.es.js.map
├── common.d.ts
├── common.es.js
├── common.es.js.map
├── generator.d.ts
├── generator.es.js
├── generator.es.js.map
├── index.d.ts
├── index.es.js
└── index.es.js.map
```

Modified the previous output type directory location

**Comment by timlrx** [comment]
Released as v1.6.3. Thanks for the fix!
